integration/kafka/src/main/java/alluxio/kafka/connect/AlluxioSinkTask.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.kafka.connect;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileSystem;
+import alluxio.exception.AlluxioException;
+import alluxio.kafka.connect.format.AlluxioFormat;
+
+import org.apache.kafka.clients.consumer.OffsetAndMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.apache.kafka.connect.sink.SinkTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class exports kafka data to Alluxio filesystem.
+ */
+public class AlluxioSinkTask extends SinkTask {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AlluxioSinkTask.class);
+
+  private AlluxioSinkConnectorConfig mConfig;
+
+  private FileSystem mFs;
+
+  private AlluxioFormat mFormat;
+
+  private Map<TopicPartition, AlluxioTopicPartitionWriter> mTpWriters =
+      new HashMap<TopicPartition, AlluxioTopicPartitionWriter>();
+
+  /**
+   * Parse the alluxio connector configuration.
+   *
+   * @param map configuration parameters
+   */
+  @Override public void start(Map<String, String> map) {
+    mConfig = new AlluxioSinkConnectorConfig(map);
+    mFs = FileSystem.Factory.get();
+    try {
+      mFormat = (AlluxioFormat) Class
+          .forName(mConfig.getString(AlluxioSinkConnectorConfig.ALLUXIO_FORMAT)).newInstance();
+    } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
+      LOG.error("Exception is {}", e);
```

Comment:
Question: should we raise the exception here, if the `mFormat` can not be initialized ?
currently, we simply `LOG` it and proceed, is this fine?